packages/hawtio/src/index.ts

```diff
@@ -12,3 +13,7 @@ export * from './ui'
 
 // Register Hawtio React component version
 configManager.addProductInfo('Hawtio React', '__PACKAGE_VERSION_PLACEHOLDER__')
+
+// Initialise a listener to update the application theme
+// depending on the browser's chosen theme.
+initWindowThemeListener()
```

Comment:
Since the theme is a UI concern, why can't we invoke it at `Hawtio` react component instead of here?

You should be able to invoke it before or after this part:
https://github.com/hawtio/hawtio-next/blob/4f29a26c14802d36023f2be9f1473423c46e2bf7/packages/hawtio/src/Hawtio.tsx#L15-L17